<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintActivitiesSameStartingTimeForm_template</class>
 <widget class="QDialog" name="ConstraintActivitiesSameStartingTimeForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the dialog to see the list of all constraints of this type">Constraints activities same starting time</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="constraintsTextLabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Please read Help/Important tips, advice 2). It is IMPORTANT to remove redundant min days constraints after adding constraints same starting time. Click the Help button!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="constraintsTextLabel_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>NOTE: You might not need this constraint. Press Help!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="constraintsTextLabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Constraints</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="constraintsListWidget"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="currentConstraintTextLabel">
       <property name="text">
        <string>Current constraint</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="currentConstraintTextEdit"/>
     </item>
     <item>
      <widget class="QGroupBox" name="filterGroupBox_2">
       <property name="title">
        <string>Filter</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="teachersTextLabel">
          <property name="text">
           <string>Teacher</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="teachersComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="studentsTextLabel">
          <property name="text">
           <string>Students set</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="studentsComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="subjectsTextLabel">
          <property name="text">
           <string>Subject</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="subjectsComboBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="activityTagsTextLabel">
          <property name="text">
           <string>Activity tag</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="activityTagsComboBox"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addConstraintPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyConstraintPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeConstraintPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
